src/index.js

```diff
@@ -309,6 +309,17 @@ const indy = {
     throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
   },
 
+  async listMyDidsWithMeta(wh) {
+    return IndySdk.listMyDidsWithMeta(wh)
+  },
+    
+  async setDidMetadata(wh, did, metadata) {
+    if (Platform.OS === 'ios') {
+      return IndySdk.setDidMetadata(metadata, did, wh)
+    }
```

Comment:
Same here, I don't think it is implemented yet

Reply:
Just added those, and it looks like they have the same functionality in the ios wrapper - albeit with a different name 

https://github.com/hyperledger/indy-sdk/blob/7f4fbb055478181761347a981544fb823d6f47ba/wrappers/ios/libindy-pod/Indy/Wrapper/IndyDid.h#L182-L193

Reply:
^ same as above. Thanks

Reply:
resolved